app/src/main/res/values/strings.xml

```diff
@@ -471,7 +471,7 @@
   <string name="topic_revision_recyclerview_tag">topic_revision_recyclerview_tag</string>
   <string name="ongoing_recycler_view_tag">ongoing_recycler_view_tag</string>
   <string name="completed_story_list_recyclerview_tag">completed_story_list_recyclerview_tag</string>
-  <string name="item_selection_text">Please select at least one choice.</string>
+  <string name="item_selection_text">Please select all correct choices.</string>
```

Comment:
Is there an existing test to verify this wording?

Reply:
Hey @BenHenning there is no test available to verify this as of now.

Reply:
@BenHenning was looking at how to write tests for this on ExplorationActivityTest, could not find a way to directly do this. Is there a way to initially create the fragment consisting of multiple choice options in order instead of single choice in order to test if the heading value matches what we expect. If tests are not necessarily needed for this PLMK.

Reply:
We have existing StateFragmentTest and StateFragmentLocalTest suites where we could add a test for this case, but given the limited time we have I'm going to resolve this & merge the PR. The benefits of a test for this change are fairly small, anyway.